geode-wan/src/distributedTest/java/org/apache/geode/internal/cache/UpdateVersionDUnitTest.java

```diff
@@ -270,8 +271,11 @@ private VersionTag putEntryAndGetReplicatedRegionVersionTag(String key) {
 
     EntryEventImpl event =
         createNewEvent((DistributedRegion) region, versionTag, entry.getKey(), "value-3");
-
-    ((LocalRegion) region).basicUpdate(event, false, true, 0L, false);
+    try {
+      ((LocalRegion) region).basicUpdate(event, false, true, 0L, false);
+    } catch (ConcurrentCacheModificationException ex) {
```

Comment:
Is this catch needed?  Shouldn't the changes be a no change for all existing callers?

Reply:
You're right, it wasn't needed, it's been removed.